sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuTextBasedPartitionReader.scala

```diff
@@ -230,6 +235,26 @@ abstract class GpuTextBasedPartitionReader(
     }
   }
 
+  def dateFormat: String
+
+  def castStringToDate(input: ColumnVector, ansiEnabled: Boolean): ColumnVector = {
```

Comment:
This looks like it is never called with ansiEnabled. Is there a reason to keep this parameter?